kahuna/public/js/search/results.js

```diff
@@ -384,6 +399,106 @@ results.controller('SearchResultsCtrl', [
             globalErrors.trigger('clipboard', sharedUrl);
         };
 
+      const validatePhotoSalesSelection = (images) => {
+        let validImages = [];
+        let invalidImages = [];
+        images.forEach( (image) => {
+          if (image.data.usages.data.length === 0) {
+            validImages.push(image);
+          } else {
+            let syndicationExists = false;
+            for (const usage of image.data.usages.data) {
+              if (usage.data.platform === "syndication") {
+                syndicationExists = true;
+                break;
+              }
+            }
+            (syndicationExists === true ? invalidImages : validImages).push(image);
+          }
+        });
+        return [validImages, invalidImages];
+      };
+
+      ctrl.sendToPhotoSales = () => {
+        try {
+          const validImages = validatePhotoSalesSelection(ctrl.selectedImages)[0];
+          validImages.map(image => {
+            mediaApi.syndicateImage(image.data.id, "Capture", "true");
+          });
+          ctrl.clearSelection();
+          const notificationEvent = new CustomEvent("newNotification", {
+            detail: {
+              announceId: "sendToCaptureNoValidImages",
```

Comment:
I'd rather see an announceId that reflects the meaning of the message (e.g. "sendToCaptureSuccess") or one that is very generic (e.g "sendToCaptureNotification") - which would be used for all notifications